src/toil/test/cwl/cwlTest.py

```diff
@@ -405,7 +405,10 @@ def test_bioconda(self):
             out_name="output1",
         )
 
+    # TODO: work out why we can't fulfil the SoftwareRequirement and it crashes
```

Comment:
`fulfil`

Though this is fixed now, correct?

Reply:
I guess it must be if it is working in mainline.